README.md

```diff
@@ -15,6 +15,13 @@ source .env
 env | grep TABIA
 ```
 
+Install required tools:
+```bash
```

Comment:
Code blocks need to be surrounded by white lines. WIll add a markdown linter to CI.